api/src/formatters/yaml-nested.ts

```diff
@@ -3,7 +3,7 @@ import { config } from '../config';
 import { Exporter, IntermediateTranslationFormat, Parser } from '../domain/formatters';
 
 export const yamlNestedParser: Parser = async (data: string) => {
-  const parsed = yaml.safeLoad(data);
+  const parsed = yaml.load(data);
```

Comment:
@rahul-rathore-576 is there "safeLoad" method now or some parameter to load() which make it "safe"? Or it's safe by default now?

Reply:
@evereq In latest version of `js-yaml` they are completely removed `safeLoad` and `safeDump` methods and use now `load` and `dump` methods instead.